src/components/editCenter/MainInformation.js

```diff
@@ -21,20 +24,42 @@ const MainInformation = ({step, setStep, clubs, cities, locations, setLocations,
     useEffect(() => {
         if (result) {
             mainInformationFrom.setFieldsValue({...result})
+            setLocations(result.locations);
         }
     }, [])
 
     const onFinish = (values) => {
-        values.locations.map(location => {
+        if (locations.length <= 0) {
+            message.info('Ви не додали жодної локації!');
+            return;
+        }
+        locations.map(location => {
             if (!location.coordinates) {
                 location.coordinates = location.latitude + ", " + location.longitude;
             }
         })
+        values.locations = locations;
         setResult(Object.assign(result, values));
+        console.log(result)
```

Comment:
Try to avoid console.log in final commit. Remove it in other changed files if exist and close each line with ;
Also check if there unused imports